From 12f3824a30300154928c3482b02b6e1e63ce8be8 Mon Sep 17 00:00:00 2001
From: Antonio Alvarez Feijoo <antonio.feijoo@suse.com>
Date: Tue, 26 Sep 2023 08:12:23 +0200
Subject: [PATCH 02/10] fix(systemd-pcrphase): rename systemd-pcrphase binary
 to systemd-pcrextend

The `systemd-pcrphase` binary has been renamed to `systemd-pcrextend` since
https://github.com/systemd/systemd/commit/32295fa0

(Cherry-picked 8d9b558939ebdbbbeefbb1cbcd6d6825efc901fb from PR#2526)
---
 modules.d/01systemd-pcrphase/module-setup.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/modules.d/01systemd-pcrphase/module-setup.sh b/modules.d/01systemd-pcrphase/module-setup.sh
index fa960a42..77f05eb7 100755
--- a/modules.d/01systemd-pcrphase/module-setup.sh
+++ b/modules.d/01systemd-pcrphase/module-setup.sh
@@ -6,7 +6,7 @@
 check() {
 
     # If the binary(s) requirements are not fulfilled the module can't be installed.
-    require_binaries "$systemdutildir"/systemd-pcrphase || return 1
+    require_binaries "$systemdutildir"/systemd-pcrextend || return 1
 
     # Return 255 to only include the module, if another module requires it.
     return 255
@@ -27,7 +27,7 @@ depends() {
 install() {
 
     inst_multiple -o \
-        "$systemdutildir"/systemd-pcrphase \
+        "$systemdutildir"/systemd-pcrextend \
         "$systemdsystemunitdir"/systemd-pcrphase-initrd.service \
         "$systemdsystemunitdir/systemd-pcrphase-initrd.service.d/*.conf" \
         "$systemdsystemunitdir"/initrd.target.wants/systemd-pcrphase-initrd.service
-- 
2.42.0

